examples/organiq-raspberry/pi-client.js

```diff
@@ -0,0 +1,29 @@
+
+//////////////////////////////////////////////////////////////////////////////
+//pi-client.js

```

Comment:
e.g.:

This is a client application that connects to and controls the Rasperry Pi in the example. It sits in a loop, reading the button state every second, and writing a message to stdout...